guides/common/assembly_provisioning-virtual-machines-kvm.adoc

```diff
@@ -20,7 +20,7 @@ Ensure no other DHCP services run on this network to avoid conflicts with {Smart
 For more information about network service configuration for {SmartProxyServer}s, see {ProvisioningDocURL}Configuring_Networking[Configuring Networking].
 ifdef::satellite[]
 * A {RHEL} server running KVM virtualization tools (libvirt daemon).
-For more information, see the https://access.redhat.com/documentation/en-US/Red_Hat_Enterprise_Linux/7/html/Virtualization_Getting_Started_Guide/[Red Hat Enterprise Linux 7 Virtualization Getting Started Guide].
+For more information, see the https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/7/html-single/virtualization_getting_started_guide/[Red Hat Enterprise Linux 7 Virtualization Getting Started Guide].
```

Comment:
It was, for more than ten years, customary in Red Hat not to link to `html-single` versions of guides. Has that policy changed?

Reply:
@swadeley if that was a pattern, it is no longer in practice, there are links to html-single across Red Hat's docs. If you are familiar with any artifacts around that line of thinking, I'd be curious to understand the motivation and if it's appropriate in future we can remediate.